flutter/ios/Classes/SentryFlutterReplayScreenshotProvider.m

```diff
@@ -0,0 +1,46 @@
+@import Sentry;
+
+#if SENTRY_TARGET_REPLAY_SUPPORTED
+#import "SentryFlutterReplayScreenshotProvider.h"
+#import <Flutter/Flutter.h>
+
+@implementation SentryFlutterReplayScreenshotProvider {
+  FlutterMethodChannel *channel;
+}
+
+- (instancetype _Nonnull)initWithChannel:
+    (FlutterMethodChannel *_Nonnull)channel {
+  if (self = [super init]) {
+    self->channel = channel;
+  }
+  return self;
+}
+
```

Comment:
curious why we need to implement screenshot provider and breadcrumb converter in objc, is it for performance?

Reply:
the types we need from cocoa are not visible outside their module so we need to do it in objc which ignores swift visibility rules.